--- gtkmm-2.4.0/gdk/gdkmm/Makefile.am.orig	2004-04-10 13:50:53.000000000 +0200
+++ gtkmm-2.4.0/gdk/gdkmm/Makefile.am	2004-04-16 08:25:42.892922544 +0200
@@ -26,5 +26,5 @@
 lib_LTLIBRARIES		= libgdkmm-2.4.la
 libgdkmm_2_4_la_SOURCES	= $(files_built_cc) $(files_extra_cc)
 libgdkmm_2_4_la_LDFLAGS	= $(common_ldflags)
-libgdkmm_2_4_la_LIBADD	= $(win32_dlls_extra_libs) $(GDKMM_LIBS)
+libgdkmm_2_4_la_LIBADD	= $(sublib_win32_dlls_libs) $(GDKMM_LIBS)
 
--- gtkmm-2.4.0/gtk/gtkmm/Makefile.am.orig	2004-04-10 13:42:16.000000000 +0200
+++ gtkmm-2.4.0/gtk/gtkmm/Makefile.am	2004-04-16 08:26:05.546478680 +0200
@@ -33,5 +33,5 @@
 lib_LTLIBRARIES		= libgtkmm-2.4.la
 libgtkmm_2_4_la_SOURCES	= $(files_built_cc) $(files_extra_cc)
 libgtkmm_2_4_la_LDFLAGS	= $(common_ldflags)
-libgtkmm_2_4_la_LIBADD	= $(win32_dlls_extra_libs) $(GTKMM_LIBS)
+libgtkmm_2_4_la_LIBADD	= $(sublib_win32_dlls_libs) $(GTKMM_LIBS)
 
