diff -aurN gtkmm-2.2.3.orig/glib/glibmm/ustring.h gtkmm-2.2.3/glib/glibmm/ustring.h
--- gtkmm-2.2.3.orig/glib/glibmm/ustring.h	2003-04-21 00:40:43.000000000 +0200
+++ gtkmm-2.2.3/glib/glibmm/ustring.h	2003-06-12 20:19:55.000000000 +0200
@@ -520,7 +520,7 @@
 
 //! @}
 
-private:
+public:
 
 #ifndef DOXYGEN_SHOULD_SKIP_THIS
 
@@ -533,6 +533,7 @@
 
 #endif /* DOXYGEN_SHOULD_SKIP_THIS */
 
+private:
   std::string string_;
 };
 
