diff -urN aaa/gtkmm-2.2.1/examples/Makefile.am_fragment gtkmm-2.2.1/examples/Makefile.am_fragment
--- aaa/gtkmm-2.2.1/examples/Makefile.am_fragment	Wed Apr  9 23:54:17 2003
+++ gtkmm-2.2.1/examples/Makefile.am_fragment	Thu Apr 10 00:36:34 2003
@@ -23,12 +23,12 @@
 
 install-example-src:
 	@$(NORMAL_INSTALL)
-	$(mkinstalldirs) $(gtkmm_docdir)/$(subdir)
+	$(mkinstalldirs) $(DESTDIR)/$(gtkmm_docdir)/$(subdir)
 	for p in $(srcdir)/*.cc $(srcdir)/*.h; do \
 	  if test -f $$p; then \
 	    f="`echo $$p | sed -e 's|^.*/||'`"; \
-	    echo " $(INSTALL_DATA) $$p $(gtkmm_docdir)/$(subdir)/$$f"; \
-	    $(INSTALL_DATA) $$p $(gtkmm_docdir)/$(subdir)/$$f; \
+	    echo " $(INSTALL_DATA) $$p $(DESTDIR)/$(gtkmm_docdir)/$(subdir)/$$f"; \
+	    $(INSTALL_DATA) $$p $(DESTDIR)/$(gtkmm_docdir)/$(subdir)/$$f; \
 	  fi; \
 	done
 
