--- gtkmm-2.2.1/configure.in.orig	Sat Mar 15 11:02:06 2003
+++ gtkmm-2.2.1/configure.in	Sat Apr 12 19:35:21 2003
@@ -22,9 +22,9 @@
 GTKMM_MICRO_VERSION=1
 GTKMM_VERSION=$GTKMM_MAJOR_VERSION.$GTKMM_MINOR_VERSION.$GTKMM_MICRO_VERSION
 GTKMM_RELEASE=$GTKMM_MAJOR_VERSION.$GTKMM_MINOR_VERSION
-AC_DEFINE_UNQUOTED(GTKMM_MAJOR_VERSION, $GTKMM_MAJOR_VERSION)
-AC_DEFINE_UNQUOTED(GTKMM_MINOR_VERSION, $GTKMM_MINOR_VERSION)
-AC_DEFINE_UNQUOTED(GTKMM_MICRO_VERSION, $GTKMM_MICRO_VERSION)
+AC_DEFINE_UNQUOTED(GTKMM_MAJOR_VERSION, $GTKMM_MAJOR_VERSION, [major version])
+AC_DEFINE_UNQUOTED(GTKMM_MINOR_VERSION, $GTKMM_MINOR_VERSION, [minor version])
+AC_DEFINE_UNQUOTED(GTKMM_MICRO_VERSION, $GTKMM_MICRO_VERSION, [micro version])
 AC_SUBST(GTKMM_VERSION)
 AC_SUBST(GTKMM_RELEASE)
 
--- gtkmm-2.2.1/scripts/macros.m4.orig	Fri Dec 27 09:42:13 2002
+++ gtkmm-2.2.1/scripts/macros.m4	Sat Apr 12 19:35:56 2003
@@ -116,7 +116,7 @@
 
   if test "x$gtkmm_debug_refcounting" = "xyes"; then
   {
-    AC_DEFINE([GTKMM_DEBUG_REFCOUNTING],[1])
+    AC_DEFINE([GTKMM_DEBUG_REFCOUNTING],[1],[debug refcounting])
   }
   fi
 ])
--- gtkmm-2.2.1/scripts/cxx_std.m4.orig	Thu Apr 25 15:26:13 2002
+++ gtkmm-2.2.1/scripts/cxx_std.m4	Sat Apr 12 19:37:48 2003
@@ -35,7 +35,7 @@
 
   if test "x${gtkmm_cv_cxx_has_namespace_std}" = "xyes"; then
   {
-    AC_DEFINE([GTKMM_HAVE_NAMESPACE_STD],[1])
+    AC_DEFINE([GTKMM_HAVE_NAMESPACE_STD],[1],[have namespace std])
   }
   fi
 ])
@@ -70,7 +70,7 @@
 
   if test "x${gtkmm_cv_cxx_has_std_iterator_traits}" = "xyes"; then
   {
-    AC_DEFINE([GTKMM_HAVE_STD_ITERATOR_TRAITS],[1])
+    AC_DEFINE([GTKMM_HAVE_STD_ITERATOR_TRAITS],[1],[have std::iterator_traits])
   }
   fi
 ])
@@ -105,7 +105,7 @@
 
   if test "x${gtkmm_cv_cxx_has_sun_reverse_iterator}" = "xyes"; then
   {
-    AC_DEFINE([GTKMM_HAVE_SUN_REVERSE_ITERATOR],[1])
+    AC_DEFINE([GTKMM_HAVE_SUN_REVERSE_ITERATOR],[1],[have Sun reverse_iterator])
   }
   fi
 ])
@@ -146,7 +146,7 @@
 
   if test "x${gtkmm_cv_cxx_has_template_sequence_ctors}" = "xyes"; then
   {
-    AC_DEFINE([GTKMM_HAVE_TEMPLATE_SEQUENCE_CTORS],[1])
+    AC_DEFINE([GTKMM_HAVE_TEMPLATE_SEQUENCE_CTORS],[1],[have templated sequence ctors])
   }
   fi
 ])
