--- gtkmm-2.2.10/scripts/cxx.m4.orig	2004-03-14 00:39:32.000000000 +0100
+++ gtkmm-2.2.10/scripts/cxx.m4	2004-03-29 21:57:55.710744944 +0200
@@ -199,7 +199,7 @@
 
   if test "x${gtkmm_cv_cxx_can_disambiguate_const_template_specializations}" = "xyes"; then
   {
-    AC_DEFINE([GLIBMM_HAVE_DISAMBIGUOUS_CONST_TEMPLATE_SPECIALIZATIONS],[1, [Defined if the compiler does not find it ambiguous to have both const and non-const template specializations]])
+    AC_DEFINE([GLIBMM_HAVE_DISAMBIGUOUS_CONST_TEMPLATE_SPECIALIZATIONS],[1], [Defined if the compiler does not find it ambiguous to have both const and non-const template specializations])
   }
   fi
 ])
